src/test/java/teammates/test/cases/action/InstructorCourseEnrollAjaxPageActionTest.java

```diff
@@ -0,0 +1,73 @@
+package teammates.test.cases.action;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.util.Const;
+import teammates.ui.controller.AjaxResult;
+import teammates.ui.controller.InstructorCourseEnrollAjaxPageAction;
+import teammates.ui.pagedata.InstructorCourseEnrollAjaxPageData;
+
+/**
+ * SUT: {@link InstructorCourseEnrollAjaxPageAction}.
+ */
+public class InstructorCourseEnrollAjaxPageActionTest extends BaseActionTest {
+    @Override
+    protected String getActionUri() {
+        return Const.ActionURIs.INSTRUCTOR_COURSE_ENROLL_AJAX_PAGE;
+    }
+
+    @Override
+    @Test
+    public void testExecuteAndPostProcess() {
+        InstructorAttributes instructor = typicalBundle.instructors.get("instructor1OfCourse1");
+        String instructorId = instructor.googleId;
+
+        gaeSimulation.loginAsInstructor(instructorId);
+        ______TS("Unsuccessful case: not enough parameters");
+        verifyAssumptionFailure();
+        String[] submissionParams = new String[] {};
+        verifyAssumptionFailure(submissionParams);
+
+        ______TS("typical successful case");
+        submissionParams = new String[] {
+                Const.ParamsNames.COURSE_ID, instructor.courseId,
+                Const.ParamsNames.COURSE_INDEX, "1",
```

Comment:
What is this for?

Reply:
This is to pass the course ID as required by `InstructorCourseEnrollAjaxPageAction`. Course ID 1 in this case refers to the Course 1 in `typicalDataBundle.json`

Reply:
Huh? I don't see any `COURSE_INDEX` in `InstructorCourseEnrollAjaxPageAction`

Reply:
Looking at the latest commit, it has been removed, but somehow still shows up here on Github

Reply:
Yup I resolved this and checked that it is not in my local build too. Not sure why it's still here.